airbyte-integrations/connectors/source-mixpanel/source_mixpanel/manifest.yaml

```diff
@@ -232,7 +252,7 @@ definitions:
         fields:
           - path:
               - browser_version
-            value: "{{ record.browser_version | string }}"
+            value: "{{ record.browser_version | string if record.browser_version else '' }}"
```

Comment:
did we see a case where this happens? It's not clear why this change is required from the PR description

Reply:
1. If we do not add conversion to string then I see records where value can be a number on our account.
2. During tests I also saw that when 'browser_version' is missing in records then value in record:
"browser_version": "{{ record.browser_version11111 | string if record.browser_version else '' }}"

So this update is needed